<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookedAtcDialog</class>
 <widget class="QDialog" name="BookedAtcDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Booked ATC</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/icons/qutescoop.png</normaloff>:/icons/qutescoop.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLineEdit" name="editFilter">
       <property name="toolTip">
        <string>wildcards (*, ?) and powerful regular expressions allowed</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Search:</string>
       </property>
       <property name="buddy">
        <cstring>editFilter</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Filter time:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="7">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>for</string>
       </property>
       <property name="buddy">
        <cstring>spinHours</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <widget class="QSpinBox" name="spinHours">
       <property name="specialValueText">
        <string>ever</string>
       </property>
       <property name="accelerated">
        <bool>true</bool>
       </property>
       <property name="suffix">
        <string> h</string>
       </property>
       <property name="maximum">
        <number>120</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QTimeEdit" name="timeFilter">
       <property name="wrapping">
        <bool>true</bool>
       </property>
       <property name="time">
        <time>
         <hour>23</hour>
         <minute>59</minute>
         <second>59</second>
        </time>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QDateEdit" name="dateFilter">
       <property name="wrapping">
        <bool>true</bool>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="timeSpec">
        <enum>Qt::UTC</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="9">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="10">
      <widget class="QToolButton" name="tbPredict">
       <property name="text">
        <string>Warp to</string>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="arrowType">
        <enum>Qt::RightArrow</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="boxResults">
     <property name="title">
      <string>Results</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTreeView" name="treeBookedAtc">
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="uniformRowHeights">
         <bool>true</bool>
        </property>
        <property name="itemsExpandable">
         <bool>false</bool>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblStatusInfo">
       <property name="text">
        <string>waiting for data</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>BookedAtcDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>557</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>602</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
